Assets/SEE/Controls/DesktopPlayerMovement.cs

```diff
@@ -22,11 +22,23 @@ private struct CameraState
 
         private CameraState cameraState;
 
+        private CharacterController controller;
+
         [Tooltip("The code city which the player is focusing on.")]
         public GO.Plane FocusedObject;
 
         private void Start()
         {
+            controller = GetComponent<CharacterController>();
+
+            // Default Layer is ignored by collider
+            controller.excludeLayers = 1;
+
+            // Defines the build in collider of the charactercontroller
+            controller.center = new Vector3(0.0f, 1.55f, 0.21f);
+            controller.radius = 0.44f;
+            controller.height = 0.0f;
```

Comment:
Are these the dimensions of the collider, i.e., the shape that can collide with other objects? If so, why are these specific values chosen for center, radius, and height? A short explanatory comment would probably help.

Reply:
Yes these are the dimensions of the collider. We want that if the player is going down only his head is in the sceen but the whole avatar stays in the sceen when going up. Also there is some puffer in front of the users face.

Reply:
I will try to adress this 👍

Reply:
It is addressed now.